discord/member.py

```diff
@@ -142,7 +142,7 @@ class Member(discord.abc.Messageable, _BaseUser):
     status : :class:`Status`
         The member's status. There is a chance that the status will be a :class:`str`
         if it is a value that is not recognised by the enumerator.
-    activity: Union[:class:`Game`, :class:`Streaming`, :class:`Activity`]
+    activity: Union[:class:`Game`, :class:`Streaming`, :class:`Activity`, :class:`Spotify`]
```

Comment:
Good, but I think that Spotify should go before Activity, as Game and Streaming are.

Reply:
Fixed